docs/rfcs/rfc-036-authorization.md

```diff
@@ -20,27 +20,28 @@ type RemoteJWKS = {
 
 type Key = string | RemoteJWKS
 
-type AuthorizationConfig = {
+type AuthConfig = {
   key: Key | ((req: RequestLike) => Key);
   verify: boolean;
   // https://github.com/panva/jose/blob/main/docs/interfaces/jwt_verify.JWTVerifyOptions.md
   verifyOptions: JWTVerifyOptions;
+  globalAuthentication: boolean;
```

Comment:
I feel like we've discussed this before but I don't remember why it was not a good idea: if we support `@authentication` could this be replaced by a schema-level `@authentication` ?

Reply:
I don't actually know how this got here, I don't recall adding that line! 😅 We should really have an RFC for the `@authentication` directive!